contracts/interfaces/IOrderMixin.sol

```diff
@@ -154,11 +156,12 @@ interface IOrderMixin {
     ) external payable returns(uint256 makingAmount, uint256 takingAmount, bytes32 orderHash);
 
     /**
-     * @notice Same as `fillOrderTo` but uses contract-based signatures.
+     * @notice Same as `fillOrder` but uses contract-based signatures.
      * @param order Order quote to fill
      * @param signature Signature to confirm quote ownership
      * @param amount Taker amount to fill
-     * @param takerTraits Specifies threshold as maximum allowed takingAmount when takingAmount is zero, otherwise specifies minimum allowed makingAmount. Top-most bit specifies whether taker wants to skip maker's permit.
+     * @param takerTraits Specifies threshold as maximum allowed takingAmount when takingAmount is zero, otherwise specifies
+     * minimum allowed makingAmount. The 2nd (0 based index) highest bit specifies whether taker wants to skip maker's permit.
```

Comment:
It seems that docgen may generate markdown text containing "\n" and "*". We need to verify and ensure that this is not the case.